DirectEmployers/MyJobs
# [PD 3158] Adding the feedback button back

## What's this PR do?
Adds the feedback button back to NUO

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the NUO app and look on the right hand side Navigation

## How should this be manually tested?
Pull down the repository. Go into the NUO application and you will see on the right hand navigation side there is the link for the beta feedback

## Any background context you want to provide?
Needed to add back since it already got merged. The removal will only take place in next sprint
